controller/embedding/manager.py

```diff
@@ -96,36 +82,25 @@ def __embed_one_by_one_helper(
 ) -> None:
     for embedding_item in embedding_data:
         splitted = embedding_item.get("name").split("-", 2)
-        if embedding_item.get("type") == enums.EmbeddingType.ON_ATTRIBUTE.value:
-            connector.request_creating_attribute_level_embedding(
-                project_id,
-                attribute_id=attribute_names[splitted[0]],
-                user_id=user_id,
-                config_string=splitted[2],
-            )
-        elif embedding_item.get("type") == enums.EmbeddingType.ON_TOKEN.value:
-            connector.request_creating_token_level_embedding(
-                project_id,
-                attribute_id=attribute_names[splitted[0]],
-                user_id=user_id,
-                config_string=splitted[2],
-            )
+        platform = project_manager.__get_platform_name(embedding_item.get("name"))
```

Comment:
even though possible, i don't think we should call "private" methods. I'd change it to a public method. (also I couldn't find the function in the gateway so maybe there is something wrong here anyway)

Reply:
outdated